README.md

```diff
@@ -93,6 +93,28 @@ Possible values are:
 * `messages`: display requests and responses notifications
 * `verbose`: display each request and response as JSON
 
+### Debugging using VS Code
+
+The `launch.json` contains a 'Minitest - current file' configuration for the debugger.
+
+1. Add a breakpoint using the VS Code UI.
+1. Open the relevant test file.
+1. Open the **Run and Debug** panel on the sidebar.
+1. Ensure `Minitest - current file` is selected in the top dropdown.
+1. Press `F5` OR click the green triangle next to the top dropdown. VS Code will then run the test file with debugger activated.
+1. When the breakpoint is triggered, the process will pause and VS Code will connect to the debugger and activate the debugger UI.
+1. Open the Debug Console view to use the debugger's REPL.
+
+```
+rm `which rdbg`
+```
+
+### Debugging using the command line
+
+You can attach to the ruby-lsp process from the command line with:
+
+`bundle exec rdbg --attach`
```

Comment:
This section is also not needed I think?
FWIW, once VS Code is connected to the debugger, the `DEBUG CONSOLE` tab can be used as a console:
![Screenshot 2023-01-11 at 18 44 25](https://user-images.githubusercontent.com/5079556/211891561-7033e6bb-aa1d-4108-8d15-3c9df1b54f85.png)